bark_ml/library_wrappers/lib_tf_agents/runners/cdqn_runner.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import logging
+import time
+import tensorflow as tf
+import numpy as np
+tf.compat.v1.enable_v2_behavior()
+
+# BARK imports
+from bark.runtime.commons.parameters import ParameterServer
+
+# tf agent imports
+from tf_agents.drivers import dynamic_step_driver
+from tf_agents.drivers import dynamic_episode_driver
+from tf_agents.metrics import tf_metrics
+from tf_agents.eval import metric_utils
+from tf_agents.utils import common
+from tf_agents.trajectories import time_step as ts
+
+# BARK-ML imports
+from bark_ml.library_wrappers.lib_tf_agents.runners.tfa_runner import TFARunner
+
+
+class CDQNRunner(TFARunner):
+  def __init__(self,
+               environment=None,
+               agent=None,
+               params=None):
+    TFARunner.__init__(self,
+                       environment=environment,
+                       agent=agent,
+                       params=params)
+
+  @tf.function
+  def _inference(self, input):
+    q_logits, _ = self.q_model.call(input)
```

Comment:
can you elaborate this more in detail and add docstrings?

Reply:
Q network of the categorical DQN returns a value distribution for each action. This function calculates the Expectation what is the q-value we want